libs/designsystem/icon/src/icon-registry.service.spec.ts

```diff
@@ -1,8 +1,12 @@
 import { IconRegistryService } from './icon-registry.service';
+import { kirbyIconSettings } from './kirby-icon-settings';
 
 describe('KirbyIconRegistryService', () => {
   let service: IconRegistryService;
-
+  const defaultIcons = kirbyIconSettings.icons;
```

Comment:
Consider assigning these variables inside `beforeEach` so the state of e.g. defaultIcons is not persisted between individual tests where we might add or remove icons.